npm/cypress-schematic/src/schematics/ng-add/files/cypress.config.ts

```diff
@@ -1,11 +1,11 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  'integrationFolder': '<%= root%>cypress/integration',
-  'supportFile': '<%= root%>cypress/support/component.ts',
-  'videosFolder': '<%= root%>cypress/videos',
-  'screenshotsFolder': '<%= root%>cypress/screenshots',
-  'pluginsFile': '<%= root%>cypress/plugins/index.ts',
-  'fixturesFolder': '<%= root%>cypress/fixtures',
   'baseUrl': '<%= baseUrl%>',
+  'e2e': {
```

Comment:
What's the reason for explicitly specifying config options that are the default values?

Reply:
I went ahead and removed them. However, this does bring up a good point about IA. I can find this on configuring CT(https://deploy-preview-4186--cypress-docs.netlify.app/guides/getting-started/component-framework-configuration), but I can't find an equivalent on E2E testing. The config reference with that info is kind of buried - https://deploy-preview-4186--cypress-docs.netlify.app/guides/references/configuration. 

IMO referencing pieces of that doc more in the framework config pieces would help users - especially new users or those with limited experience configuring Cypress.